docs/backends/akv.md

```diff
@@ -0,0 +1,86 @@
+## Azure Key Vault secrets
+
+#### Prerequisites
```

Comment:
Is there a way to create this via the Azure CLI? Might be better to document like that, since the UI tends to change more over time.